runtime/stdlib/account.go

```diff
@@ -2615,6 +2615,21 @@ func getCapabilityController(
 			newStorageCapabilityControllerRetargetFunction(inter, address, controller)
 		controller.DeleteFunction =
 			newStorageCapabilityControllerDeleteFunction(inter, address, controller)
+		controller.SetTagFunction = interpreter.NewHostFunctionValue(
```

Comment:
New

Reply:
Maybe refactor this into a function, cf. `newStorageCapabilityControllerDeleteFunction`